app/Models/Forum/Post.php

```diff
@@ -233,6 +233,14 @@ public function isValid()
             }
         }
 
+        $trimmedBody = trim($this->body_raw);
+        while (preg_match('/^\[quote(?:=".+?")?\].+?\[\/quote\]/s', $trimmedBody, $matches)) {
```

Comment:
This doesn't quote match nested quotes right:

```[quote="i"][quote="am"][quote="a"]teapot[/quote][/quote][/quote]```
matches
```[quote="i"][quote="am"][quote="a"]teapot[/quote]```

Reply:
or maybe just use `BBCodeFromDB::removeBlockQuotes`?

Reply:
yeah I should be just using that. but now upon checking, it doesn't seem to work either in all cases